sale_order_country_allowed_product/tests/test_sale_order_country_allowed_product.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2025 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.exceptions import ValidationError
+from odoo.tests.common import Form, TransactionCase
+
+
+class TestSaleOrderCountryAllowedProduct(TransactionCase):
+    def setUp(self):
```

Comment:
"@classmethod" is not necessary  ?

Reply:
It is not, as we are using setUp and not setUpClass. There's only one test, so I think that it is the same using setUp and setUpClass.

Reply:
This is Odoo 16.0 which means that we should use setUpClass instead of setUp. This improves the performance significantly. Check the [migration wiki](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0) for details.

![image](https://github.com/user-attachments/assets/4ca91471-ae94-4fb8-899b-176f545f4f8b)